File: src/main/java/com/ebuozturk/ecommerce/dto/CreateCartRequest.java
package com.ebuozturk.ecommerce.dto;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.util.Date;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class CreateCartRequest {
    private String name;
    private String no;
    private Date expiryDate;
    private String cvc;
    private Long userId;
}

File: src/main/java/com/ebuozturk/ecommerce/dto/cart/CreateCartRequest.kt
package com.ebuozturk.ecommerce.dto.cart;

import java.util.*

data class CreateCartRequest @JvmOverloads constructor(
    val name: String,
    val no: String,
    val expiryDate: Date,
    val cvc: String,
    val userId: String
){
}
